README.md

```diff
@@ -67,3 +67,62 @@ void loop() {
   delay(1000);
 }
 ```
+### Usage with two SHT31 sensors
+
+```c++
+#include <Wire.h>
+#include "SHTSensor.h"
+
+// Sensor with normal i2c address
+// Sensor 1 with address pin pulled to GND
+SHTSensor sht1(SHTSensor::SHT3X);
+
+// Sensor with alternative i2c address
+//Sensor 2 with address pin pulled to Vdd
+SHTSensor sht2(SHTSensor::SHT3X_ALT);
+
+void setup() {
+  // put your setup code here, to run once:
+  Wire.begin();
+  Serial.begin(9600);
+  delay(1000); // let serial console settle
+  // init will always be successful
```

Comment:
@abrauchli should init() maybe call readSample() in any case, not just in the autodetect one, to ensure there's a sensor listening?

Reply:
Hi @winkj, I thought about it for a while and I'm happy to take your opinion as I'm open to both options. So far I came up with manually specifying the sensor type is already an advanced use case, as such it's OK - as long as documented - to not actually check for a sensor during `init`.